tests/unit/test_error_handling.py

```
@@ -0,0 +1,398 @@
+                MessagePassing(aggregator=invalid_agg)
+
+            with pytest.raises(ValueError, match="Invalid aggregator"):
+                GINConv(output_dim=16, aggregator=invalid_agg)
+
+            with pytest.raises(ValueError, match="Invalid aggregator"):
+                SAGEConv(output_dim=16, aggregator=invalid_agg)
+
+    def test_invalid_input_shapes(self):
+        """
+        Tests that GCNConv raises appropriate errors when given inputs with invalid shapes.
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)